### PR TITLE
Train 299 [1.9]

### PR DESCRIPTION
**Pull request by colin-msphere**
* #2227 more robust handling of zk pid
* #2251 bump mesos package

### CODE REVIEW

**Comment by colin-msphere** [comment]
@mesosphere-mergebot bump-ee

File: packages/mesos/buildinfo.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "7fdb5cb179a15a63240590a2c3e0a6cdbdab89b5",
-    "ref_origin" : "dcos-mesos-1.2.x-f8706e5"
+    "ref": "bd983e54c7d941db4a67c1d5659a60014b3fec71",
+    "ref_origin" : "dcos-mesos-1.2.x-13a0782"
```
Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: Mesos bump PR needs an update, moved back to WIP. Please remove from the train.

**Comment by colin-msphere** [comment]
@mesosphere-mergebot bump-ee

**Comment by lingmann** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm https://jira.mesosphere.com/browse/DCOS-19925

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**